.ibm/pipelines/env_variables.sh

```diff
@@ -47,6 +47,8 @@ GH_RHDH_QE_USER_TOKEN=$(cat /tmp/secrets/GH_RHDH_QE_USER_TOKEN)
 
 GITLAB_TOKEN=$(cat /tmp/secrets/GITLAB_TOKEN)
 
+RHDH_PR_OS_CLUSTER_URL=$(cat /tmp/secrets/RHDH_PR_OS_CLUSTER_URL)
```

Comment:
This is needed to do the `oc login` when sending test results from a cluster not connected to the RH network.